### ISSUE TITLE
[BUG] DataFrame.skew and DataFrame.kurtosis tests are breaking

### ISSUE THREAD

**Issue by trxcllnt** [rapidsai/node-rapids#245]
```
@rapidsai/cudf: FAIL test/cudf-data-frame-tests.ts (35.981 s)
@rapidsai/cudf:   ● dataframe.kurtosis
@rapidsai/cudf:     TypeError: this.get(...).kurtosis is not a function
@rapidsai/cudf:       1603 |    */
@rapidsai/cudf:       1604 |   kurtosis<P extends keyof T>(skipna = true) {
@rapidsai/cudf:     > 1605 |     const result = this.names.map((name) => { return (this.get(name) as any).kurtosis(skipna); });
@rapidsai/cudf:            |                                                                              ^
@rapidsai/cudf:       1606 |     return Series.new(result) as any as Series < T[P] extends Numeric ? Numeric : never > ;
@rapidsai/cudf:       1607 |   }
@rapidsai/cudf:       1608 |
@rapidsai/cudf:       at src/data_frame.ts:1605:78
@rapidsai/cudf:           at Array.map (<anonymous>)
@rapidsai/cudf:       at DataFrame.kurtosis (src/data_frame.ts:1605:31)
@rapidsai/cudf:       at Object.<anonymous> (test/cudf-data-frame-tests.ts:585:46)
@rapidsai/cudf:   ● dataframe.skew
@rapidsai/cudf:     TypeError: this.get(...).skew is not a function
@rapidsai/cudf:       1625 |    */
@rapidsai/cudf:       1626 |   skew<P extends keyof T>(skipna = true) {
@rapidsai/cudf:     > 1627 |     const result = this.names.map((name) => { return (this.get(name) as any).skew(skipna); });
@rapidsai/cudf:            |                                                                              ^
@rapidsai/cudf:       1628 |     return Series.new(result) as any as Series < T[P] extends Numeric ? Numeric : never > ;
@rapidsai/cudf:       1629 |   }
@rapidsai/cudf:       1630 |
@rapidsai/cudf:       at src/data_frame.ts:1627:78
@rapidsai/cudf:           at Array.map (<anonymous>)
@rapidsai/cudf:       at DataFrame.skew (src/data_frame.ts:1627:31)
@rapidsai/cudf:       at Object.<anonymous> (test/cudf-data-frame-tests.ts:595:42)
```

**Comment by mzegar** [comment]
Forgot to wrap this in a `expect.ToThrow()` and I guess forgot to run tests...

**Comment by mzegar** [comment]
Nevermind, I forgot I actually fixed this issue in this PR :+1: 
https://github.com/rapidsai/node-rapids/pull/240/files

**Comment by mzegar** [comment]
FIxed in #240
